cli/test/lib/tasks/install_spec.ts

```diff
@@ -345,7 +347,7 @@ describe('/lib/tasks/install', function () {
         beforeEach(function () {
           sinon.stub(util, 'isInstalledGlobally').returns(true)
 
-          state.getBinaryPkgAsync.resolves({ version: 'x.x.x' })
+          ;(state.getBinaryPkgAsync as any).resolves({ version: 'x.x.x' })
```

Comment:
same

Reply:
personally I'm fine with it until we either improve the types or migrate the testing harness. The other sane option with spec files at least is to just us an `@ts-expect-error` as we get a lot of typing errors when we writing tests. something like:

```typescript
// @ts-expect-error
state.getBinaryPkgAsync.resolves({ version: 'x.x.x' })
```

is completely normal